specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -378,6 +533,10 @@ export const userPreferenceDefinitions = {
             renderer: WelcomePageModePreferenceItem,
             container: 'div',
           }),
+          /*
```

Comment:
This is not merged correctly. This comment was removed once https://github.com/specify/specify7/issues/2999 was fixed, but got added back during merge conflict resolution